docs/src/pages/components/data-grid/export/export.md

```diff
@@ -7,13 +7,17 @@ components: DataGrid, XGrid
 
 <p class="description">Easily export the rows in various file formats such as CSV, Excel, or PDF.</p>
 
-## 🚧 CSV export
+## CSV export
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #197](https://github.com/mui-org/material-ui-x/issues/197) if you want to see it land faster.
+You are able to export the displayed data to CSV with an API call, or using the grid UI.
+
+### Export selector
```

Comment:
Do you envision more h3 headers in the near future? Right now, it doesn't seem to add anything as it's the only item.

Reply:
I wanted to mention the component somewhere because we have kind of the same thing for the density. I can remove it and basically explain it as part of the CSV Export itself.

Another option and how I initially made it is to have it as `## Export selector` above the CSV Export

Reply:
If we don't plan to add more sections for the CSV export, I think that a single h2 is enough.